src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/query.svelte

```diff
@@ -45,14 +56,17 @@
     </div>
   {:then types}
     <div class="flex items-center gap-4">
-      <Select label="Query Type" value={queryType}>
+      <Select label="Query Type" bind:value={queryType}>
         {#each types as value}
           <Option {value}>{value}</Option>
         {/each}
       </Select>
+      <Button on:click={() => query(queryType)} icon={faRedo} loading={isLoading}>
```

Comment:
We've got a promise here anyway, I wonder if we could just use the `#await` syntax in Svelte.